packages/python-packages/doc-warden/warden/WardenConfiguration.py

```diff
@@ -42,7 +42,13 @@ def __init__(self):
             '-d',
             '--scan-directory',
             dest = 'scan_directory',
-            help = 'The repo directory that this tool should be scanning.',
+            help = 'The directory on a repo that this tool should be scanning.',
+            required = True)
+        parser.add_argument(
+            '-h',
+            '--repo-root',
+            dest = 'repo_root',
+            help = 'The root of the repo',
```

Comment:
This makes repo_root a required argument, this is technically a breaking change.

Reply:
I made it optional now. it will be set to the target_directory (scan_directory) if it si not set.